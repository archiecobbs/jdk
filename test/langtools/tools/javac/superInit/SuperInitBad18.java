/*
 * @test /nodynamiccopyright/
 * @bug 8194743
 * @summary Verify super() can't appear in an initialization block
 *
 * @compile/fail/ref=SuperInitBad18.out -XDrawDiagnostics SuperInitBad18.java
 */
import java.util.concurrent.atomic.AtomicReference;
public class SuperInitBad18 extends AtomicReference<Object> {
    SuperInitBad18() {
        super(new Object() {
            {
                super();
            }
        });
    }
}
