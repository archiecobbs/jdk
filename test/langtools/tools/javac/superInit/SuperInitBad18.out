SuperInitBad18.java:13:22: compiler.err.call.must.only.appear.in.ctor: super
1 error
